### ISSUE TITLE
[Accessibility] Make breadcrumbs accessible

### ISSUE THREAD

**Issue by andresgalante** [auth0/cosmos#1289]
Breadcrumbs are not accessible. To fix them:

- Make the breadcrumb container a `nav` with an `aria-label=Breadcrumb`
- the trail of links should be an ordered list `ol`
- the active link should have a `aria-current=page`
- ideally, the separator should not be a DOM element so it doesn't get announced.

Source: https://www.w3.org/TR/wai-aria-practices/#breadcrumb

Also:

- Write accessibility docs

And:

- why is the "backlink" a breadcrumb?
